addons/website/static/src/scss/website.scss

```diff
@@ -428,6 +435,14 @@ font[class*='bg-'] {
     padding: 2px 6px 4px;
 }
 
+.lead {
+    $-bs-lead-ratio: 1.2; // Bootstrap '.lead' font-size multiplier.
```

Comment:
Since we use this variable's **value** in`bootstrap_overridden.scss`, we should move its declaration in `primary_variable.scss` and use it for the  `$lead-font-size` computation as well.